x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/file/FileUserPasswdStoreTests.java

```diff
@@ -103,6 +103,19 @@ public void testStore_AutoReload() throws Exception {
 
         watcherService.start();
 
+        try (BufferedWriter writer = Files.newBufferedWriter(file, StandardCharsets.UTF_8, StandardOpenOption.APPEND)) {
+            writer.append("\n");
+        }
+
+        if (latch.await(200, TimeUnit.MILLISECONDS)) {
+            fail("Listener should not be called as users passwords are not changed.");
+        }
```

Comment:
Although we do have a bunch of them, we'd like to avoid having tests that have arbitrary time based waits in them. Those 200ms start to add up when we have a lot of tests.

For the scenario tested here, I'd be happy with a test that explicitly called `watcherService.notifyNow(ResourceWatcherService.Frequency.HIGH)` to explicitly trigger a file reload so that we don't need to wait.